modules/nw-sriov-configuring-device.adoc

```diff
@@ -83,9 +79,12 @@ A device can be used in either mode.
 ====
 
 [start=2]
-. Create the CR by running the following command:
+. Create the SriovNetworkNodePolicy CR. Replace `<name>` with the name for this configuration.
 +
 ----
-$ oc create -f <filename> <1>
+$ oc create -f <name>-sriov-node-network.yaml
 ----
-<1>  Replace `<filename>` with the name of the file you created in the previous step.
+
+After the applying the configuration update, all the Pods in `sriov-network-operator` namespace transition to the `Running` status.
+
+// TODO - How do I verify success?
```

Comment:
This can be verified by checking the `SriovNetworkNodeState` for the selected nodes.

Reply:
@zshi-redhat, silly question, but what is a good command for this? I've been attempting iterations with `oc`, but as I don't have any SR-IOV hardware locally, I don't know if I've come up with anything that is useful for this.

Thanks.

Reply:
Good question, I was to trying it myself :-)

Here is an example to get the status of applying a SriovNetworkNodePolicy:
`oc get sriovnetworknodestates.sriovnetwork.openshift.io -n openshift-sriov-network-operator nfvpe-08.oot.lab.eng.bos.redhat.com -o jsonpath='{.status.syncStatus}'`

`sriovnetworknodestates.sriovnetwork.openshift.io` is the CRD name of SriovNetworkNodeState
`nfvpe-08.oot.lab.eng.bos.redhat.com` is a node name (same to the name of SriovNetworkNodeState CR for that node).
`openshift-sriov-network-operator` is the namespace of sriov network operator

So a general format of above cmd is:
``oc get sriovnetworknodestates.sriovnetwork.openshift.io -n openshift-sriov-network-operator <node-name> -o jsonpath='{.status.syncStatus}'``

Reply:
@zshi-redhat, updated!